src/Nethermind/Nethermind.Specs/GoerliSpecProvider.cs

```diff
@@ -44,9 +44,9 @@ public void UpdateMergeTransitionInfo(long? blockNumber, UInt256? terminalTotalD
     }
 
     public ulong NetworkId => BlockchainIds.Goerli;
-    public ulong ChainId => NetworkId;
+    public ulong ChainId => BlockchainIds.Goerli;
```

Comment:
Shouldn't we remove Goerli BlockchainIds?

Reply:
> fixes #6876
> 
> ## Changes
> * Removes internal support for Goerli network
> * Removes references to Goerli from the code base
> * Leaves and Isolates GoerliSPecProvider, GoerliSPecProviderTest and CliqueBlockProducerTest since they are useful 

> #### Notes on testing
> We need CliqueBlockProducerTest to test nethermind for clique behaviour and that test depends on GoerliSPecProvider which in turn is tested by GoerliSPecProviderTest so all three files and two tests have to stay unmodified. They do not depend on any external config or configuration files.
> 
> Some other tests that didn't really need goerli where modified to account for goerli being phased out.

@MarekM25 
So we need it to for these other classes that would remain.
Also, since the ID is reserved forever it done't hurt to have it in the open.
Could just add a comment to say end of life exceeded or post end of life.